assets/libraries/openapi/library.rego

```diff
@@ -1,9 +1,13 @@
 package generic.openapi
 
-checkOpenAPI(doc) = version {
+check_openapi(doc) = version {
   object.get(doc, "openapi", "undefined") != "undefined"
   version = doc.openapi
   regex.match("^3\\.0\\.\\d+$", doc.openapi)
 } else = version {
   version = "undefined"
 }
+
+is_valid_url(url){
+    regex.match(`(https?):\/\/([\w_-]+(?:(?:\.[\w_-]+)+))([\w.,@?^=%&:\/~+#-]*[\w@?^=%&\/~+#-])?`, url)
```

Comment:
This regex allows urls like `http://example.com#@evil.com/x.txt`, which must be invalid